sim/dex.ts

```diff
@@ -363,14 +363,26 @@ export class ModdedDex {
 		}
 		if (!this.data.Pokedex.hasOwnProperty(id)) {
 			let aliasTo = '';
-			if (id.startsWith('mega') && this.data.Pokedex[id.slice(4) + 'mega']) {
-				aliasTo = id.slice(4) + 'mega';
-			} else if (id.startsWith('m') && this.data.Pokedex[id.slice(1) + 'mega']) {
-				aliasTo = id.slice(1) + 'mega';
-			} else if (id.startsWith('primal') && this.data.Pokedex[id.slice(6) + 'primal']) {
-				aliasTo = id.slice(6) + 'primal';
-			} else if (id.startsWith('p') && this.data.Pokedex[id.slice(1) + 'primal']) {
-				aliasTo = id.slice(1) + 'primal';
+			const formeNames = {
+				a: 'alola'
+				alolan: 'alola',
+				g: 'galar',
+				galarian: 'galar',
+				gigantamax: 'gmax',
+				gmax: 'gmax',
```

Comment:
gmax === gmax ???

Reply:
wrote that for startsWith
currently if you use `/dt gmax charizard` it wont work